ts/navigation/MainNavigator.tsx

```diff
@@ -44,16 +45,34 @@ const getIcon = (routeName: string): string => {
   return route === undefined ? fallbackIcon : route;
 };
 
-const styles = StyleSheet.create({
-  tabBarStyle: {
-    height: 64,
+const tabBarStyleFactory = () => {
+  const defaultStyle = {
     backgroundColor: variables.colorWhite,
     paddingLeft: 3,
     paddingRight: 3,
     borderTopWidth: 0,
-    paddingTop: 8,
-    paddingBottom: 10
-  },
+    paddingTop: 8
+  };
+
+  // Add space to the bottom bar for iPhone 12
+  const isIPhone12 = deviceInfoModule.getDeviceId().indexOf("iPhone13") !== -1;
```

Comment:
```suggestion
  const isIPhone12 = deviceInfoModule.getDeviceId().indexOf("iPhone12") !== -1;
```

Did you mean `iPhone12`?

Reply:
No, the `deviceId`s for the iPhone 12 product line are "13,1"; "13,2" ... See https://gist.github.com/adamawolf/3048717, also confirmed through `console.log`. Added comment for this.